addons/map/ACE_Settings.hpp

```diff
@@ -17,6 +17,12 @@ class ACE_Settings {
         displayName = CSTRING(BFT_HideAiGroups_DisplayName);
         description = CSTRING(BFT_HideAiGroups_Description);
     };
+    class GVAR(BFT_ShowPlayerNames) {
```

Comment:
None of the settings have, but it could be a great oportunity for adding them.

@looterz, could you add this to all the settings?

`category = CSTRING(Module_DisplayName);`